Sheheroes_WatchOS/pubspec.lock

```diff
@@ -0,0 +1,852 @@
+# Generated by pub
+# See https://dart.dev/tools/pub/glossary#lockfile
```

Comment:
Please remove this lock file from commit, but keep this in your local.
See Why: https://dart.dev/guides/libraries/private-files#pubspeclock